travis-ci/docker-compose.yml

```diff
@@ -18,6 +18,7 @@ services:
     volumes:
       - ../packages/test-data:/usr/local/apache2/htdocs:ro
   sftp:
+    # image: panubo/sshd:latest
```

Comment:
Seems like we can remove this change?

Reply:
I kept it in as a reminder that the current option is temporary, but I'm not really attached to it.

Reply:
OK. I'm fine either way